code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -741,3 +744,44 @@ mob/living/simple_animal/bullet_act(var/obj/item/projectile/Proj)
 				var/matrix/M = new()
 				B.transform = M.Scale(scale)
 				B.update_icon()
+
+/mob/living/simple_animal/proc/rename_self_helper(mob/M, regex/name_pattern, name_message, bad_name_message)
+	var/input = input(M, name_message)
```

Comment:
Wrap this into a `input = sanitizeName(input)` call. Otherwise we'll have really bad names.

Reply:
> Wrap this into a `input = sanitizeName(input)` call. Otherwise we'll have really bad names.

The regex guidelines set up perform the duty of that call while allowing later customization if desired in a future prompt.  Double stacking sanitize on top of it is unnecessary,

Reply:
I mean. Technically correct. Tho this doesn't check against the size limit, for example.